src/components/Accordion/AccordionItemDefer.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from 'react';
+import { AccordionItem } from 'carbon-components-react';
+
+const AccordionItemDefer = ({ children, ...props }) => {
+  const { open } = props;
+  const [openState, setOpenState] = useState(open);
+  const [hasEverBeenOpened, setHasEverBeenOpened] = useState(open);
+  useEffect(
+    () => {
+      setOpenState(open);
+    },
+    [open]
+  );
+  const handleToggle = event => {
+    if (props.onHeadingClick) {
+      props.onHeadingClick(event);
+    }
+    setOpenState(!openState);
+    setHasEverBeenOpened(true);
+  };
+  return (
+    <AccordionItem data-testid="accordion-item-deferred" {...props} onHeadingClick={handleToggle}>
```

Comment:
Should we be adding the `iot` prefix for data-testid's?

Reply:
We haven't established that pattern as they're inherently scoped to our library, but we could if you think that would be helpful!

Reply:
Sounds good. I was thinking it could clash with others if they're using `data-testid` as well